README.md

```diff
@@ -2,78 +2,71 @@
 
 ![Build Status](https://travis-ci.org/Pitt-CSC/PittAPI.svg?branch=master)
```

Comment:
Also, it doesn't look like we use Travis anymore, so is it OK to remove this badge?
```suggestion
```

Reply:
yeah, I think so? might need Alex or Ritwik to comment just to be sure

Reply:
Go for it, though someone should probably set up Travis again

Reply:
Replaced the Travis badge with a GitHub Actions badge